src/common/hive_partitioning.cpp

```diff
@@ -316,6 +316,14 @@ std::map<idx_t, const HivePartitionKey *> HivePartitionedColumnData::GetReverseM
 	return ret;
 }
 
+std::map<idx_t, const HivePartitionKey *> GlobalHivePartitionState::GetReverseMap() {
```

Comment:
Can this be a `const_reference<HivePartitionKey>` instead?

Reply:
This was not needed, so removed, pointer is still there in `HivePartitionedColumnData::GetReverseMap()`, but that I will leave unmodified.